### PR TITLE
0.9.0 release docs

### PR DESCRIPTION
**Pull request by oicheryl**
This should be all of the remaining changes for 0.9.0.

### CODE REVIEW

File: _docs/install/docker/container.md [hunk 1]
```diff
@@ -42,7 +35,7 @@ docker run -d --name storageos \
     --device /dev/fuse \
     -v /var/lib/storageos:/var/lib/storageos:rshared \
     -v /run/docker/plugins:/run/docker/plugins \
-    storageos/node server
+    storageos/node:0.9 server
```
Comment (croomes) [inline _docs/install/docker/container.md]: Should be 0.9.0

File: _docs/install/docker/plugin.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ token]({%link _docs/install/prerequisites/clusterdiscovery.md %}) with
 `JOIN` when you install the container:
 
 ```bash
-$ docker plugin install --alias storageos storageos/plugin ADVERTISE_IP=xxx.xxx.xxx.xxx JOIN=xxxxxxxxxxxxxxxxx
+$ docker plugin install --alias storageos storageos/plugin:0.9 ADVERTISE_IP=xxx.xxx.xxx.xxx JOIN=xxxxxxxxxxxxxxxxx
```
Comment (croomes) [inline _docs/install/docker/plugin.md]: 0.9.0

File: _docs/install/prerequisites/firewalls.md [hunk 3]
```diff
@@ -11,80 +11,74 @@ StorageOS requires a few ports to be open in order to function correctly. These
...
-ufw allow 4223/tcp
-ufw allow 8222/tcp
-ufw allow 13701/tcp
-ufw allow 13701/udp
-ufw allow 17100/tcp
-ufw allow 17102/tcp
+ufw allow 5708/tcp
+ufw allow 5709/tcp
+ufw allow 5710/tcp
+ufw allow 5711/tcp
```
Comment (croomes) [inline _docs/install/prerequisites/firewalls.md]: and udp:

```
ufw allow 5711/udp
```

(I haven't tested this though)

File: _docs/install/prerequisites/firewalls.md [hunk 4]
```diff
@@ -11,80 +11,74 @@ StorageOS requires a few ports to be open in order to function correctly. These
...
+iptables -A INPUT -p tcp --dport 5708 -j ACCEPT
+iptables -A OUTPUT -p tcp --dport 5708 -j ACCEPT
+iptables -A INPUT -p tcp --dport 5708 -j ACCEPT
+iptables -A OUTPUT -p tcp --dport 5708 -j ACCEPT
+iptables -A INPUT -p tcp --dport 5709 -j ACCEPT
+iptables -A OUTPUT -p tcp --dport 5709 -j ACCEPT
+iptables -A INPUT -p tcp --dport 5710 -j ACCEPT
+iptables -A OUTPUT -p tcp --dport 5710 -j ACCEPT
+iptables -A INPUT -p tcp --dport 5711 -j ACCEPT
+iptables -A OUTPUT -p tcp --dport 5711 -j ACCEPT
```
Comment (croomes) [inline _docs/install/prerequisites/firewalls.md]: add:
```
iptables -A OUTPUT -p udp --dport 5711 -j ACCEPT
```
again, not tested

File: assets/Vagrantfile [hunk 5]
```diff
@@ -16,10 +15,10 @@ docker run -d --name storageos \
     --device /dev/fuse \
     -v /var/lib/storageos:/var/lib/storageos:rshared \
     -v /run/docker/plugins:/run/docker/plugins \
-    storageos/node server
+    storageos/node:0.9 server
```
Comment (croomes) [inline assets/Vagrantfile]: 0.9.0

File: _docs/install/docker/container.md (hunk 1 above)
Comment (oicheryl) [reply _docs/install/docker/container.md]: Done.

File: _docs/install/docker/plugin.md (hunk 2 above)
Comment (oicheryl) [reply _docs/install/docker/plugin.md]: Done.

File: _docs/install/prerequisites/firewalls.md (hunk 3 above)
Comment (oicheryl) [reply _docs/install/prerequisites/firewalls.md]: Done.

File: _docs/install/prerequisites/firewalls.md (hunk 4 above)
Comment (oicheryl) [reply _docs/install/prerequisites/firewalls.md]: Done.

File: assets/Vagrantfile (hunk 5 above)
Comment (oicheryl) [reply assets/Vagrantfile]: Done.

**Pull request merged**